src/ui/digital-exchange/components/SearchBarContainer.js

```diff
@@ -1,15 +1,22 @@
 import { connect } from 'react-redux';
 import { filterBySearch } from 'state/digital-exchange/actions';
 import SearchBar from 'ui/digital-exchange/components/SearchBar';
+import { formValueSelector, submit } from 'redux-form';
 
+const formSel = formValueSelector('searchBar');
+
+export const mapStateToProps = state => ({
+  isFilled: !!formSel(state, 'keyword'),
+});
 
 export const mapDispatchToProps = dispatch => ({
-  onSubmit: (value) => {
-    dispatch(filterBySearch(value));
+  clearSearch: () => dispatch(submit('searchBar')),
```

Comment:
instead of dispatching the `submit` redux-form action, I suggest you to perform the reset in the `SearchBar` component, using the `reset()` method from the props (see https://redux-form.com/8.2.0/docs/api/props.md/)